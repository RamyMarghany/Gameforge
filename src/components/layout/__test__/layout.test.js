import React from 'react';
import ReactDOM from 'react-dom'
import Wrapper from './../index'
import "@testing-library/jest-dom"; 
import renderer from 'react-test-renderer'

it('render without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Wrapper></Wrapper>, div)
})


it('matches snapshots', ()=>{
    const tree = renderer.create(<Wrapper/>).toJSON()
    expect(tree).toMatchSnapshot();
})