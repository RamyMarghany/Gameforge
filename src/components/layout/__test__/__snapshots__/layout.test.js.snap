// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<section
  className="stylecss__BodyWrapper-sc-1kagjb5-0 gLnaXI"
  data-testid="wrapper"
>
  <div
    className="stylecss__TabsButtonsWrapper-vczw1e-1 iGKTJa"
  >
    <button
      className="stylecss__TabsButton-vczw1e-2 jrbmlw tab-btn btnActive"
      onClick={[Function]}
    >
      Users
    </button>
    <button
      className="stylecss__TabsButton-vczw1e-2 jrbmlw tab-btn"
      onClick={[Function]}
    >
      Todos
    </button>
    <button
      className="stylecss__TabsButton-vczw1e-2 jrbmlw tab-btn"
      onClick={[Function]}
    >
      Posts
    </button>
  </div>
  <section
    className="stylecss__TabContainer-vczw1e-0 hZbfXB tab-wrapper"
  >
    <div
      className="stylecss__TabsContent-vczw1e-3 gfjuXg tabs-content"
    >
      <div
        className="tab-active"
      >
        <div
          className="stylecss__TabLayout-vczw1e-4 chNah"
        >
          <p
            className="stylecss__Loading-vczw1e-5 jXIxLR"
          >
            Loading the all users...!
          </p>
        </div>
      </div>
      <div
        className="tab"
      >
        <div
          className="stylecss__TabLayout-vczw1e-4 chNah"
        >
          <p
            className="stylecss__Loading-vczw1e-5 jXIxLR"
          >
            Loading the first 
            <b>
              20
            </b>
             todo items...!
          </p>
        </div>
      </div>
      <div
        className="tab"
      >
        <div
          className="stylecss__TabLayout-vczw1e-4 chNah"
        >
          <p
            className="stylecss__Loading-vczw1e-5 jXIxLR"
          >
            Loading the first 
            <b>
              20
            </b>
             posts...!
          </p>
        </div>
      </div>
    </div>
  </section>
  <div
    className="accordion-container"
    data-testid="accordion"
  >
    <div
      className="stylecss__AccordionTitle-sc-1jkeuws-0 kZrPQQ accordion-title"
      onClick={[Function]}
    >
      Post
    </div>
    <div
      className="stylecss__AccordionBody-sc-1jkeuws-1 drDpFH accordion-body"
    />
  </div>
  <div
    className="accordion-container"
    data-testid="accordion"
  >
    <div
      className="stylecss__AccordionTitle-sc-1jkeuws-0 kZrPQQ accordion-title"
      onClick={[Function]}
    >
      Albums
    </div>
    <div
      className="stylecss__AccordionBody-sc-1jkeuws-1 drDpFH accordion-body"
    />
  </div>
  <div
    className="accordion-container"
    data-testid="accordion"
  >
    <div
      className="stylecss__AccordionTitle-sc-1jkeuws-0 kZrPQQ accordion-title"
      onClick={[Function]}
    >
      Comments
    </div>
    <div
      className="stylecss__AccordionBody-sc-1jkeuws-1 drDpFH accordion-body"
    />
  </div>
</section>
`;
